config/templates/cloud-config-controller

```diff
@@ -693,6 +705,66 @@ write_files:
            $kubectl uncordon "$hostname"
            echo "done."'
 
+  {{if .Experimental.Plugins.Rbac.Enabled }}
+
+  #https://github.com/coreos/kube-aws/issues/230#issuecomment-274946933
+  - path: /srv/kubernetes/rbac/clusterrole.yaml
+    content: |
+        kind: ClusterRole
+        apiVersion: rbac.authorization.k8s.io/v1alpha1
+        metadata:
+            name: full-access
+        rules:
+          - apiGroups: ["*"]
+            resources: ["*"]
+            verbs: ["*"]
+          - nonResourceURLs: ["*"]
+            verbs: ["*"]
+
```

Comment:
@whereisaaron I renamed the full-access to cluster-admin, so we make this cluster role compatible with k8s < 1.5.1.